invokeai/app/invocations/latent.py

```diff
@@ -402,18 +448,42 @@ class Config(InvocationConfig):
             },
         }
 
+    def prep_mask_tensor(self, mask, context, lantents):
+        if mask is None:
+            return None
+
+        mask_image = context.services.images.get_pil_image(mask.image_name)
+        if mask_image.mode != "L":
+            # FIXME: why do we get passed an RGB image here? We can only use single-channel.
+            mask_image = mask_image.convert("L")
+        mask_tensor = image_resized_to_grid_as_tensor(mask_image, normalize=False)
+        if mask_tensor.dim() == 3:
+            mask_tensor = mask_tensor.unsqueeze(0)
+        mask_tensor = tv_resize(
+            mask_tensor, lantents.shape[-2:], T.InterpolationMode.BILINEAR
+        )
```

Comment:
Are we sure that bilinear is the best option? Since this is just scaling the mask down, I guess it doesn't *really* matter.

Reply:
If you know what better I open to suggestions)
This code - copy from old inpaint generator code.

Reply:
I'm not at all sure, but yeah, if this is scaling it down to 1/8th size, I think it's going to be hard to notice a difference here.